README.md

```diff
@@ -58,6 +58,15 @@ use Mix.Config
 config :stripity_stripe, api_key: "YOUR SECRET KEY"
 ```
 
+For security purposes it's possible to use a function or a tuple to resolve
+the secret:
+
+```ex
+config :stripity_stripe, api_key: {MyApp.Secrets, :stripe_secret, []}
+# OR
+config :stripity_stripe, api_key: fn -> System.get_env("STRIPE_SECRET") end
```

Comment:
This seems like a great idea!  Would you mind elaborating on the security concerns?  (more curious than anything ;))

Reply:
Good that you pointed that out. 

I put "security" there because I was tired (I did this PR in a hurry) and directly translated a term from Croatian to English. Security in this context is a loaded term so I removed it in `4722133`. 

This PR only provides more ways to do configuration - in the sense that it provides "dynamic" configuration options. I need this feature because a project of mine moved to OTP releases on Kubernetes, it's easier to configure ENV variables in such an environment than to manage and mount a `config/prod.exs` (as I don't know on which machine my container will end up, and I don't want to bake the secrets into the container image).

TL;DR - this isn't security related as much as it's ease-of-use in a containerized environment related.